file-formats/scp1/type/zif_aff_scp1_v1.intf.abap

```diff
@@ -0,0 +1,300 @@
+      "! BC Set is assigned to a Software Component. If the Software Component is not installed in the system,
+      "! then activation of BC Set is aborted
+      software_component TYPE c LENGTH 30,
+      "! <p class="shorttext">Minimum Release</p>
+      "! To activate the BC Set successfully, installed Software Component must be atleast at this Release
+      "! $default {@link zif_aff_scp1_v1.data:co_validity.all_releases}
+      minimum_release    TYPE ty_release,
+      "! <p class="shorttext">Maximum Release</p>
+      "! To activate the BC Set successfully, installed Software Component must be within this Release
+      "! $default {@link zif_aff_scp1_v1.data:co_validity.all_releases}
+      maximum_release    TYPE ty_release,
+    END OF ty_attributes.
+
+  "! <p class="shorttext">Customizing Object Type</p>
+  TYPES ty_object_type TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Allowed Customizing Object Types</p>
+    BEGIN OF co_object_type,
+      "! <p class="shorttext">Viewcluster</p>
+      "! A view cluster is a group of maintenance dialogs which are collected in one maintenance unit
+      view_cluster                  TYPE ty_object_type VALUE 'C',
+      "! <p class="shorttext">Logical Transport Object</p>
+      "! Different tables can be grouped together as logical transport objects. Always entire object is exported,
+      "! not only explicitly specified table key
+      logical_transport_object      TYPE ty_object_type VALUE 'L',
+      "! <p class="shorttext">Table (with Text Table)</p>
+      "! All tables must have a generated table maintenance dialog
+      table_with_text_table         TYPE ty_object_type VALUE 'S',
+      "! <p class="shorttext">Individual Transaction Object</p>
+      "! Different tables can be grouped together as individual transport objects.
+      "! Only explicitly specified table keys are exported
+      individual_transaction_object TYPE ty_object_type VALUE 'T',
+      "! <p class="shorttext">View</p>
+      "! All maintenance views must have a generated table maintenance dialog
+      view                          TYPE ty_object_type VALUE 'V',
+    END OF co_object_type.
+
+  "! <p class="shorttext">Record is incomplete</p>
+  TYPES ty_incomplete TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Record is incomplete</p>
+    BEGIN OF co_incomplete,
+      "! <p class="shorttext">Record is complete</p>
+      "! BC Sets created using Transport or any API, puts all fields of a data record or data records.
+      "! Such data records are flagged as complete
+      complete   TYPE ty_incomplete VALUE ' ',
+      "! <p class="shorttext">Record is incomplete</p>
+      "! Automatic Customizing Recording only puts the key fields of a data record or
+      "! data records, in the BC Set. Such BC Sets must be post-processed.
+      "! Such data records are flagged as incomplete
+      incomplete TYPE ty_incomplete VALUE 'U',
+    END OF co_incomplete.
```

Comment:
Is this type used? Below it was a boolean value

Reply:
Yes, I removed the type and constant declaration.